app/src/main/java/net/squanchy/eventdetails/EventDetailsActivity.java

```diff
@@ -19,5 +35,27 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
+
+        EventDetailsComponent component = EventDetailsInjector.obtain(this);
+        service = component.service();
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+
+        Intent intent = getIntent();
+        int dayId = intent.getIntExtra(EXTRA_DAY, -1);
+        int eventId = intent.getIntExtra(EXTRA_EVENT_ID, -1);
+        subscription = service.event(dayId, eventId)
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(event -> Toast.makeText(this, "Event: " + event, Toast.LENGTH_LONG).show());
```

Comment:
I would extract to method just because less diff when you'll implement the view part, but meh